### PR TITLE
Operations added Date cells

### PR DESCRIPTION
**Pull request by DaanVermunt**
I'm allready making a pull request for this branch even though the operations aren't ready yet, but i added the Date option to the table and don't want to create conflict in other branches.

### CODE REVIEW

**Comment by Saffie** [comment]
Checkstyle: good job making attributes final when applicable

**Comment by Saffie** [comment]
Great idea creating an Enum for aggregate functions

File: health/src/main/java/com/health/operations/Chunk.java [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+	 * 
+	 * @param table
+	 * @param operations
+	 *            a map of columns and their aggreagate operation.
+	 * @param column
+	 *            on which the data is chunked with the same string.
+	 * @return
+	 */
+	public static Table chunkByString(final Table table, final String column,
+			final Map<String, AggregateFunctions> operations) {
```
Comment (LizzyScholten) [inline health/src/main/java/com/health/operations/Chunk.java]: Wouldn't it be better to subdivide this large method into several smaller methods?

**Comment by LizzyScholten** [comment]
Looks good! It only misses tests for chuncking and the aggregator class. But it is fine for now, since it is more about the Date cells.

**Pull request merged**